### DISCUSSION TITLE
Kata 3. Testeando con números aleatorios.

### DISCUSSION THREAD

**Discussion by JoseOlayo** [LaunchX-InnovaccionVirtual/CursoIntroPython#104] [Ideas]
Hola, les comparto esta implementación que hice al código de la Kata 3. Para testear con números aleatorios.
![randomnumbers](https://user-images.githubusercontent.com/99063988/153527903-65f3a176-e312-45e6-955c-8873d071eb5e.png)

Para cambiar los números obtenidos solo cambien la seed.
De esta manera no se les acabaran las ideas de números para testear.

Ojalá les sea de utilidad, no solo en está si no también en futuras ocasiones. Saludos 👋🏽

**Comment by OscarPedroza** [comment]
Muchas gracias por el aporte compañero.
